src/main/java/lotto/domain/ProfitCalculator.java

```diff
@@ -0,0 +1,29 @@
+package lotto.domain;
+
+import java.util.Map;
+import lotto.service.PurchaseService;
+
+public class ProfitCalculator {
+    private final int PERCENTAGE = 100;
+
+    public double calculateProfitRateInPercentage(Map<Rank, Integer> result) {
+        long usedMoney = calculateUsedAmountOfMoney(result);
+        long earnedMoney = calculateEarnedMoney(result);
+        return (double) earnedMoney / usedMoney * PERCENTAGE;
```

Comment:
```suggestion
        return 100.0 * earnedMoney / usedMoney;
```
PERCENTAGE 대신 UNIT을 쓰면 모르겠지만, 백분율의 100은 의미와 사용되는 이유가 명확해 매직 넘버가 아니며, 다른 숫자로 대체될 가능성이 현저히 낮습니다. 따라서 100을 그대로 적는 게 좋을 것 같아요.

Reply:
모든 변수의 네이밍을 따질 때에는 범용성도 신경써야겠네요. 많이 배웁니다!